lib/functoria/package.mli

```diff
@@ -18,10 +18,17 @@
 
 (** Representation of opam packages. *)
 
+type kind = [ `Global | `Local ]
+(** The kind of package installation:
+
+    - Global: installed with opam.
+    - Local: locally fetched to build with dune. *)
+
 type t
 (** The type for opam packages. *)
 
 val v :
+  ?kind:kind ->
```

Comment:
I'd probably name this "scope" instead of "kind" (as I still find the local/global distinction a bit confusing), but that's super minor.